cdci_data_analysis/pytest_fixtures.py

```diff
@@ -774,7 +774,7 @@ def clone_gitlab_repo(repository_url, repo_dir=None, renku_gitlab_ssh_key_path=N
     if branch_name is None:
         branch_name = 'master'
 
-    git_ssh_cmd = f'ssh -i {renku_gitlab_ssh_key_path}'
+    git_ssh_cmd = f'ssh -i {renku_gitlab_ssh_key_path} -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no'
```

Comment:
could you just remove this function entirely and only import it?

Reply:
oh yes I was planning to do it, forgot